Bottom: 0368f77f26b4b41252408036e4de5b47bd3cd09a
Top:    427d196997e60a11ae1c27cf491bbd1b65f5e4c4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-21 11:51:13 -0500

Add new option for maven configuration

All other repository types had an option to disable configuration.



---
diff --git a/roles/nexus3/tasks/main.yml b/roles/nexus3/tasks/main.yml
index 64d8c88d0..1cfa71113 100644
--- a/roles/nexus3/tasks/main.yml
+++ b/roles/nexus3/tasks/main.yml
@@ -74,14 +74,16 @@
   include: nexus-restore.yml
   when: nexus_restore_point is defined
 
-- include: create_repo_maven_proxy_each.yml
-  with_items: "{{ nexus_repos_maven_proxy }}"
+- block:
+    - include: create_repo_maven_proxy_each.yml
+      with_items: "{{ nexus_repos_maven_proxy }}"
 
-- include: create_repo_maven_hosted_each.yml
-  with_items: "{{ nexus_repos_maven_hosted }}"
+    - include: create_repo_maven_hosted_each.yml
+      with_items: "{{ nexus_repos_maven_hosted }}"
 
-- include: create_repo_maven_group_each.yml
-  with_items: "{{ nexus_repos_maven_group }}"
+    - include: create_repo_maven_group_each.yml
+      with_items: "{{ nexus_repos_maven_group }}"
+  when: nexus_config_maven
 
 - block:
     - include: create_repo_docker_hosted_each.yml
